modules/contact-form/grunion-contact-form.php

```diff
@@ -198,10 +198,12 @@ function process_form_submission() {
 
 			// Get shortcode from post meta
 			$shortcode = get_post_meta( $_POST['contact-form-id'], '_g_feedback_shortcode', true );
+			// Get the attributes from post meta. $attributes not available here
+			$atts = get_post_meta( $_POST['contact-form-id'], '_g_feedback_shortcode_atts', true );
 
 			// Format it
 			if ( $shortcode != '' ) {
-				$shortcode = '[contact-form]' . $shortcode . '[/contact-form]';
+				$shortcode = '[contact-form'.$atts.']' . $shortcode . '[/contact-form]';
```

Comment:
Missing spaces:
lines 206, 978, 980 between quotes and concatenation operator.
line 976 inside foreach brackets
Also, extra indentation in 977